python/lsst/ts/analysis/notebooks/analysisTools.py

```diff
@@ -0,0 +1,399 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.visualization import ZScaleInterval
+from lsst.daf import butler as dafButler
+
+from matplotlib import rcParams
+
+rcParams["ytick.labelsize"] = 15
+rcParams["xtick.labelsize"] = 15
+rcParams["axes.labelsize"] = 20
+rcParams["axes.linewidth"] = 2
+rcParams["font.size"] = 15
+rcParams["axes.titlesize"] = 18
+
+# common functions for AOS analysis
+
+
+def plotZernikeImageFromPickle(
```

Comment:
Method names should be `snake_case`, can you rename this `plot_zernike_image_from_pickle`?

Reply:
I was following the current  https://github.com/lsst-ts/ts_analysis_notebooks/blob/develop/python/lsst/ts/analysis/notebooks/aos/eigenZernicke.py and https://github.com/lsst-ts/ts_analysis_notebooks/blob/develop/python/lsst/ts/analysis/notebooks/aos/plotUtils.py which both use camelCase for modules as well as variables. Should they also be changed?

Reply:
Or here https://github.com/lsst-ts/ts_wep/blob/develop/python/lsst/ts/wep/Utility.py all methods are camelCase. Iim quite confused. Do I need to be changing every method and variable to snake_case, or is camelCase fine?